projects/gov-ui/src/lib/components/hmcts-error-summary/hmcts-error-summary.component.ts

```diff
@@ -0,0 +1,61 @@
+import {AfterViewInit, Component, Inject, Input, OnChanges, OnInit, SimpleChanges} from '@angular/core';
+import {DOCUMENT} from '@angular/common';
+/*
+  Error Summary component
+  State Less component
+  @property errorMessages that is array of messages.
+  Component is also responsible for scrolling. Up and Down the page when user click on links
+*/
+@Component({
+  selector: 'app-hmcts-error-summary',
```

Comment:
Can you move this html to separate html file rather than inline html?

Reply:
this will be done at the later stage as all library component has that style